svc-systems/README.md

```diff
@@ -934,8 +934,7 @@ curl -i -X POST \
 }
 ```
 
-
-
+<br>**NOTE:**<br> Created volume will be reflected only after the system resets. To see the change, [reset the computer system](#resetting-a-computer-system).
```

Comment:
individual addition/deletion will be triggered after reboot. In the meanwhile we cannot add or delete volumes.

Reply:
updated